### PR TITLE
Fs 3031 flag unscored sections

### PR DESCRIPTION
**Pull request by gio-karunakaran**
show scored unscored sections for flagging NSTF assessment, these changes need to be removed/ reworked once assessment store starts using the new fund-store-db.

### CODE REVIEW

File: app/assess/templates/flag_application.html [hunk 1]
```diff
@@ -92,6 +92,34 @@ <h1 class="govuk-heading govuk-heading-l">{{ pageHeading }}</h1>
...
+                                    'classes': 'govuk-fieldset__legend--s'
+                                    }
+                                },
+                                'attributes': {'data-qa': 'select_unscored_section_to_flag'},
+                                'items': items,
+                                'classes': 'govuk-checkboxes--small'
+                                })
+                            }}
+                        {% else %}
+                        <!-- ^^^ Remove once assessment_store uses fund-store db ^^^ -->
```
Comment (RamuniN) [inline app/assess/templates/flag_application.html:122]:
```suggestion
                        <!-- vvv Remove once assessment_store uses fund-store db vvv -->
                        {% if state.fund_name == "Night Shelter Transformation Fund" %}
                            {% set unscored_heading = '<h1 class="govuk-fieldset__heading">Unscored</h1>
                                                        <br>
                                                        <h1 class="govuk-fieldset__heading">Your Organisation</h1>' %}
                        {% else %}
                            {% set unscored_heading = '<h1 class="govuk-fieldset__heading">Unscored</h1>' %}
                        {% endif %}
                        <!-- ^^^ Remove once assessment_store uses fund-store db ^^^ -->
                            {{ govukCheckboxes({
                                'idPrefix': 'Unscored',
                                'name': form.section.id,
                                'fieldset': {
                                    'legend': {
                                        'html': unscored_heading,
                                    'classes': 'govuk-fieldset__legend--s'
                                    }
                                },
                                'attributes': {'data-qa': 'select_unscored_section_to_flag'},
                                'items': items,
                                'classes': 'govuk-checkboxes--small'
                                })
                            }}
```

**Comment by RamuniN** [comment]
May be change to `{% if section.name in ['Unscored', 'Your organisation'] %}` on line [88](https://github.com/communitiesuk/funding-service-design-assessment/pull/279/files#diff-66d69f13169e5a9c04a34a01e68b191c1ad24c3731c62d8d3496cbe980d109b8R88)

File: app/assess/templates/flag_application.html (hunk 1 above)
Comment (gio-karunakaran) [reply app/assess/templates/flag_application.html:122]: have un-committed the changes as these caused failing tests that are not worth the time to solve to outweigh the benefits these changes introduce

File: app/assess/templates/flag_application.html [hunk 2]
```diff
@@ -132,38 +161,36 @@ <h1 class="govuk-heading govuk-heading-l">{{ pageHeading }}</h1>
...
-                                        'html': "<h1 class=\"govuk-fieldset__heading\">{}</h1>".format(section["name"]),
-                                        'classes': 'govuk-fieldset__legend--s'
-                                        }
-                                    },
-                                    'attributes': {'data-qa': 'select_scored_section_to_flag'},
-                                    'items': items,
-                                    'classes': 'govuk-checkboxes--small'
-                                    })
-                                }}
+                        {# Get list of sub-sections from Scored section #}
```
[the remark this answers is not in the document]
Comment (gio-karunakaran) [reply app/assess/templates/flag_application.html:164]: ahh this is not being deleted but just moved lower down, github just marking it as deleted but from the diff we can see it's just moved below 👍🏾

**Review by RamuniN — APPROVED** [review]
LGTM

**Pull request merged**